layer-publisher/tests/e2e/utils/e2eUtils.ts

```diff
@@ -0,0 +1,98 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+/** 
```

Comment:
Are these the same files that we have in [`packages/commons/tests/utils/e2eUtils.ts`](https://github.com/awslabs/aws-lambda-powertools-typescript/tree/main/packages/commons/tests/utils)?

If they are the same, and it's possible, should we use these instead?

Reply:
it's not exposed :/